src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/send_location_request.cc

```diff
@@ -117,19 +117,6 @@ void SendLocationRequest::Run() {
     return;
   }
 
-  if (msg_params.keyExists(strings::location_image)) {
-    mobile_apis::Result::eType verification_result =
-        mobile_apis::Result::SUCCESS;
-    verification_result = MessageHelper::VerifyImage(
```

Comment:
@ashwink11 VerifyImage checks not only if image exist on FS. It also perform some other related to image checks. If is static or dynamic and if it allowed to use in there. ot sure if it should be deleted within implementation of this proposal.